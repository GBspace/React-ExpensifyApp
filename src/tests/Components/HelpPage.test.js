import React from 'react';
import {shallow} from 'enzyme';
import HelpPage from '../../Components/HelpPage';

test('should display help page correctly',()=>{
    const wrapper = shallow(<HelpPage />);
    expect(wrapper).toMatchSnapshot();
});