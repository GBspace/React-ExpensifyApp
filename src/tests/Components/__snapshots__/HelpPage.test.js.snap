// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display help page correctly 1`] = `
<div>
  Help Page
</div>
`;
